index.html

```diff
@@ -4,92 +4,88 @@
-                <span class =" Auther__Idntity--Name">John  
-                 <span class="Auther__Idntity--Family">Smith
-                  </span>
-                   </span>
-            </div>
-           <div class="Auther__Discribtion">nec ullamcorper augue sagittis.Prion Faucibus Lorem nisl,Donec commodo leo est,<br>
-                  <span class="Auther__Discribtion--Lastline">eleifend urna pellentesque eget 
-                    </span>
-             </div>
+                <span class =" Auther__Idntity--Name">John</span>
+                 <span class="Auther__Idntity--Family">Smith</span>
+           </div>
+           <div class="Content__Discribtion">nec ullamcorper augue sagittis.Prion Faucibus Lorem nisl,Donec commodo leo est,
+           </div>
+           <div class="Content__Discribtion--Lastline">eleifend urna pellentesque eget</div>
             <div class="Auther__Contact">
                   Prion scelerisque odio eu nisi viverra,nec ullamcorper gue 
-                  <a class="ContactInfo__Email" href="contactme@mysite.com" target="_blank">&ensp;contact me</a>
+                  <a class="ContactInfo__Email" href="contactme@mysite.com" target="_blank"> contact me</a>
             </div>
-            <div class="Auther__Progress">
-                  <div class="Auther__Progress--photoshop" >
-                      <label for="file">&emsp;&ensp;PHOTOSHOP<br></label> 
-                      <progress id="file" value="98" max="100"> 98% </progress>
-                      <br><span>&emsp;&emsp;&emsp;98%</span>
+            <div class="Content__Progress">
+                  <div class="Content__Progress--Photoshop" >
+                      <label class="Photoshop--lable" for="file">PHOTOSHOP<br></label> 
+                      <progress  id="file" value="98" max="100"> 98% </progress>
+                      <br><span class="Photoshop--percentge" > 98%</span>
                    </div>          
-                  <div class="Auther__Progress--ILLUSTRATOR" >
-                      <label for="file1">&emsp;&emsp;&ensp;ILLUSTRATOR<br></label>
+                  <div class="Content__Progress--ILLUSTRATOR" >
+                      <label class="ILLUSTRATOR--lable"  for="file1">ILLUSTRATOR<br></label>
                       <progress id="file1" value="70" max="100"> 70% </progress>
-                      <br><span>&emsp;&emsp;&emsp;70%</span>
+                      <br><span class="ILLUSTRATOR--percentge" >70%</span>
                    </div>
-                 <div class="Auther__Progress--HTML" >
-                      <label for="file2">&emsp;&emsp; &ensp;HTML/CSS<br></label>
+                 <div class="Content__Progress--HTML" >
+                      <label class="HTML--lable" for="file2">HTML/CSS<br></label>
                       <progress id="file2" value="60" max="100"> 60% </progress>
-                      <br><span>&emsp;&emsp;&emsp;60%</span>
+                      <br><span class="HTML--percentge">60%</span>

```

Comment:
Please use the same naming convention, 
first letter is capital the rest are small, and apply this to all class names, don't forget to update css and sass class names